components/authentication/quality-dashboard.yaml

```diff
@@ -0,0 +1,22 @@
+kind: RoleBinding
```

Comment:
Do we need this rolebinding? I don't see pipeline defined for dashboard... Maybe we could remove this rolebinding and add it in case we onboard build of dashboard here.

Reply:
On second thought... I see that we need to create secret for postgre... Maybe we should create role just for that?

Reply:
new role is created